ValheimExportHelper/Resources/RestoreShadersOnSceneLoad.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using System.IO.Compression;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+
+[InitializeOnLoad]
+public class RestoreShadersOnSceneLoad : MonoBehaviour
+{
+    public static void RestoreOriginalShaders() {
+        if (Directory.Exists("Assets/Shader"))
+        {
+            if(Directory.EnumerateFiles("Assets/Shader").ToList().Count >=1)
+            {
+                Directory.Delete("Assets/Shader", recursive: true);
+            }
+        }

```

Comment:
I don't think the other `.shader` files should be touched. I think this deletes all of them which is not what we want, it should only delete/override the ones from the zip.

Reply:
Current deployment setup does indeed clear out the current Shader/ directory of all existing .shader files

Puts them into the orig shader zip

Then dumps the shaders back into Assemblies/

I presumed this to be by design

Reply:
im missing 
63bc407b4dacba9a5f0bb38b052733ad32330755

Reply:
need to implement these changes into my refactor